Gordon360/Utils/ImageUtils.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.IO;
+
+namespace Gordon360.Utils
+{
+    public class ImageUtils : IImageUtils
+    {
+        /// <summary>
+        /// Takes a filepath for an image, navigates to it, collects the raw data
+        /// of the file and converts it to base64 format. 
+        /// 
+        /// The base64 data will not include the first part of a base64 image ("data:image/...").
+        /// This is because this part is removed in every image before being submitted, and it
+        /// is readded in the frontend before being displayed.
+        /// 
+        /// This helper function does not perform any error checking; every place that calls it
+        /// checks that the path is not empty. Theoretically if it isn't empty it's certainly a valid path.
```

Comment:
These are not guaranteed, they are consequences of the way the code is now. The documentation shouldn't list things that happen to be true.
```suggestion
```